yew/src/virtual_dom/vtag.rs

```diff
@@ -1090,26 +1075,26 @@ mod tests {
         let string_ref = &string;
         assert_class(html! { <p class=str /> }, "some-class");
         assert_class(html! { <p class=string.clone() /> }, "some-class");
-        assert_class(html! { <p class=&Some(str) /> }, "some-class");
+        assert_class(html! { <p class=(&Some(str)) /> }, "some-class");
         assert_class(html! { <p class=string_ref /> }, "some-class");
-        assert_class(html! { <p class=Some(str) /> }, "some-class");
-        assert_class(html! { <p class=Some(string.clone()) /> }, "some-class");
-        assert_class(html! { <p class=Some(string_ref) /> }, "some-class");
-        assert_class(html! { <p class=&Some(string.clone()) /> }, "some-class");
-        assert_class(html! { <p class=&Some(string_ref) /> }, "some-class");
+        assert_class(html! { <p class=(Some(str)) /> }, "some-class");
+        assert_class(html! { <p class=(Some(string.clone())) /> }, "some-class");
+        assert_class(html! { <p class=(Some(string_ref)) /> }, "some-class");
+        assert_class(html! { <p class=(&Some(string.clone())) /> }, "some-class");
+        assert_class(html! { <p class=(&Some(string_ref)) /> }, "some-class");
```

Comment:
This can now be written `class?=Some(string)`.

I won't modify this test because it is suppose to test with the parenthesis but obviously it works too since class is just a normal attribute here.